pkg/vulnsrc/vulnerability/const.go

```diff
@@ -28,6 +28,7 @@ const (
 	OSV                   types.SourceID = "osv"
 	Wolfi                 types.SourceID = "wolfi"
 	Chainguard            types.SourceID = "chainguard"
+	BitnamiVulndb         types.SourceID = "bitnami"
```

Comment:
I'm not sure whether this is the best source id. I mean, idk whether we should use `bitnami-vulndb` or sth like this.